tests/test_commands.py

```diff
@@ -56,6 +58,23 @@ def test_add_institution_command_invalid_inputs(self):
         self.assertRaisesRegex(CommandError, "exists", call_command,
                 "add_institution", "test", "https://www.foo.bar")
 
+    def test_add_institution_command_valid_inputs_update(self):
+        """
+        Ensures command works as expected for valid inputs
+        """
+        call_command("add_institution", "Test", "https://www.example.com",
+            "--update-existing")
+        call_command("add_institution", "Other Inst", "http://fed.foobar.edu/")
+        call_command("add_institution", "Test", "https://fed.university.edu/",
+             "--update-existing")
+        actual = Institution.objects.order_by("slug")
+        expected = [
+                Institution(name="Other Inst", slug="other-inst",
+                    cas_server_url="http://fed.foobar.edu/"),
+                Institution(name="Test", slug="test",
+                    cas_server_url="https://fed.university.edu/"),
+        ]
+        self._check_institutions(actual, expected)
```

Comment:
Rename to test_add_institution_command_update_existing_valid_inputs and move under the valid_inputs test.